include/measurement_kit/portable/sys/time.h

```diff
@@ -4,7 +4,20 @@
 #ifndef MEASUREMENT_KIT_PORTABLE_SYS_TIME_H
 #define MEASUREMENT_KIT_PORTABLE_SYS_TIME_H
 
-#ifndef _WIN32
+#ifdef _WIN32
+#include <measurement_kit/portable/_windows.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+int gettimeofday(struct timeval *tvp, void *tzp);
```

Comment:
Here we should explain what is going on on Windows, i.e. why we need to declare the prototype